### DISCUSSION TITLE
Toaros on vm

### DISCUSSION THREAD

**Discussion by ghost** [klange/toaruos#269] [Q&A]
I want to try out Toaros on a virtual machine and need your help

**Comment by ghost** [comment]
also is there a way to dual boot?

**Comment by klange** [comment]
Instructions for running ToaruOS in various emulators are listed in the README: https://github.com/klange/toaruos#running-toaruos

CD images are available from releases: https://github.com/klange/toaruos/releases

ToaruOS does not offer any installation method, and the only supported way to run the OS is from a CD. "Dual booting" is irrelevant.

**Reply by ghost** [reply]
thanks
